packages/notebook-extension/schema/tracker.json

```diff
@@ -694,8 +694,8 @@
       "default": "code"
     },
     "autoStartDefaultKernel": {
-      "title": "Automatically Start Preferred Kernel",
-      "description": "Whether to automatically start the preferred kernel.",
+      "title": "Automatically Start Selected Kernel",
+      "description": "Whether to automatically start the Selected kernel.",
```

Comment:
When user is in the Settings window, the "Selected kernel" is not clear because they do not have a way to make a selection in Settings. We need to describe when the selection happens.

Reply:
@krassowski I went ahead and updated the description, making it clear where the user can make a kernel selection.